apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -48,6 +48,13 @@ const ProgressBubbleSet = React.createClass({
     disabled: PropTypes.bool.isRequired
   },
 
+  getIcon(level) {
+    if (level.icon === null){
+      level.icon = "fa-desktop";
+    }
+    return level.icon.substring(3);
+  },
```

Comment:
I bet we could add a helper that could be used both here and in ProgressLevelSet.jsx. Stealing the code from that location, it would look something like this.
```js
function getIconForLevel(level) {
    if (level.icon) {
      // Eventually I'd like to have dashboard return an icon type. For now, I'm just
      // going to treat the css class it sends as a type, and map it to an icon name.
      const match = /fa-(.*)/.exec(level.icon);
      if (!match || !match[1]) {
        throw new Error('Unknown iconType: ' + level.icon);
      }
      return match[1];
    }

    return 'desktop';
}
```
The right place for this to live is probably progressRedux.js

Then, here we can just do:
```js
// beginning of file
import { getIconForLevel } from '@cdo/apps/code-studio/progressRedux';
// ...
// on line 82:
levelIcon={getIconForLevel(level)}